Source/ProgressionSystemRuntime/Private/ProgressionSystemDataAsset.cpp

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#include "ProgressionSystemDataAsset.h"
+
+#include <string>
+
+#include UE_INLINE_GENERATED_CPP_BY_NAME(ProgressionSystemDataAsset)
```

Comment:
Please remove this empty DataAsset .cpp file

Reply:
![image](https://github.com/h4rdmol/ProgressionSystem/assets/5227233/711766f3-91e1-46e6-9e6d-be6749de094e)
UE throws a warning regarding this one, so I've added it.  Shall I remove it according to your comment?